MdeModulePkg/MdeModulePkg.dec

```diff
@@ -471,6 +471,9 @@
   ## Include/Guid/VariableRuntimeCacheInfo.h
   gEdkiiVariableRuntimeCacheInfoHobGuid = { 0x0f472f7d, 0x6713, 0x4915, { 0x96, 0x14, 0x5d, 0xda, 0x28, 0x40, 0x10, 0x56 }}
 
+  # GUID used for ACPI Info Hob.
+  gAcpiTableHobGuid = { 0xf9886b57, 0x8a35, 0x455e, { 0xbb, 0xb1, 0x14, 0x65, 0x5e, 0x7b, 0xe7, 0xec }}

```

Comment:
The comments should be updated to specify the hob data format, because this hob definition is not defined in public header file.